src/store/useWordsStore.ts

```diff
@@ -0,0 +1,17 @@
+import { create } from 'zustand';
+import { WordQuestionType } from '@/pages/GamePage/GameWord/GameWord';
+
+interface I_UseWordsStore {
+  wordsStore: WordQuestionType;
+  setWordStore: (wordsAll: WordQuestionType) => void;
```

Comment:
```suggestion
  wordsStore: WordQuestionType;
  setWordsStore: (wordsAll: WordQuestionType) => void;
```
useWordStore에 wordStore는 없어서 setWordStore에 s가 빠진 것 같아요! 👍